_posts/2020-01-22-cockpit-211.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Cockpit 211
+author: mvo
+date: 2020-01-22
+tags: cockpit linux
+slug: cockpit-211
+category: release
+summary: Cockpit with certificate improvements and more
```

Comment:
maybe "with certificate improvements and layout fixes"?

Reply:
Yes.